src/components/cc-logs/cc-logs.stories.js

```diff
@@ -0,0 +1,558 @@
+    }
+
+    function isStarted () {
+      return timer != null;
+    }
+
+    function start (rate) {
+      if (isStarted()) {
+        stop();
+      }
+
+      timer = setInterval(add, rate);
+      refresh();
+    }
+
+    function stop () {
+      if (isStarted()) {
+        clearInterval(timer);
+        timer = null;
+      }
+      refresh();
+    }
+
+    function toggleStartStop () {
+      if (isStarted()) {
+        stop();
+      }
+      else {
+        start(rate);
+      }
+    }
+
+    function onRateToggle ({ detail }) {
+      rate = parseInt(detail);
+      refresh();
+      start(rate);
+    }
+
+    function onStartStopClick () {
+      toggleStartStop();
+    }
+
+    function onAddClick () {
+      range(1, 1).forEach(add);
+    }
+
+    function onClearClick () {
+      clear();
+    }
+
+    function onLimitChanged ({ detail }) {
+      limit = detail;
+      refresh();
+    }
+
+    function onScrollDown () {
+      container.querySelector('cc-logs').scrollToBottom();
+    }
+
+    function onFollowSwitched () {
+      follow = !follow;
+      refresh();
+    }
+
+    function onFollowChanged (e) {
+      follow = e.detail;
+      refresh();
+    }
+
+    function refresh () {
+      render(template(), container);
+    }
+
+    function template () {
+      const started = isStarted();
+
+      return html`
+        <div class="ctrl">
+          <cc-toggle .value=${`${rate}`} @cc-toggle:input=${onRateToggle} .choices=${RATES}></cc-toggle>
+          <cc-button
+            @cc-button:click=${onStartStopClick}
+            ?circle=${true}
+            ?hide-text=${true}
+            ?danger=${started}
+            ?success=${!started}
+            .icon=${started ? stopIcon : playIcon}
+            .accessibleName=${started ? 'Stop' : 'Run'}
+          ></cc-button>
+          <cc-button
+            @cc-button:click=${onAddClick}
+            ?circle=${true}
+            ?hide-text=${true}
+            ?primary=${true}
+            ?outlined=${true}
+            .icon=${addIcon}
+            accessible-name="Add one log"
+          ></cc-button>
+          <cc-button
+            @cc-button:click=${onClearClick}
+            ?circle=${true}
+            ?hide-text=${true}
+            ?danger=${true}
+            ?outlined=${true}
+            .icon=${clearIcon}
+            accessible-name="Clear"
+          ></cc-button>
+          
+          <cc-input-number
+            @cc-input-number:input=${onLimitChanged}
+            .value=${limit}
+            label="Limit"
+            min="0"
+            controls
+            inline
+          ></cc-input-number>
+
+          <cc-toggle legend="Predicate" .value=${`${predicate}`} @cc-toggle:input=${onPredicateToggle} .choices=${PREDICATES} inline></cc-toggle>
+          
+          <div class="spacer"></div>
+
+          <label for="follow">
+            <input id="follow" type="checkbox" @change=${onFollowSwitched} .checked=${follow}> Follow
```

Comment:
**💭 thought:** This control worked randomly during my testing.

Reproducing steps:
- start fake logs by choosing rabbit mode
- toggle the "Follow" checkbox
- cc-logs leave follow mode as expected
- toggle the "Follow" checkbox again
- :arrow_up: random behaviors start occurring : sometimes toggling doesn't work, sometimes it does but cc-logs follow mode get desync, sometimes it works

Reply:
fixed. Now in sandbox